### PR TITLE
feat(Stake.us): add presence

### PR DESCRIPTION
**Pull request by Kevin-Kwan**
## Description 

Add PreMiD presence for the online social casino Stake.us (United States version of Stake.com, still owned by Stake). This displays browsing different parts of the site, games currently being played, and current balance alongside selected currency for play. It is different from the existing Stake.com presence as Stake.us uses different currencies, lacks sports betting, and has many slot games missing (most likely due to US regulation of online casinos?). Thus, this presence is mainly for US users of Stake.us.

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</summary>
</details>

### CODE REVIEW

**Comment by Kevin-Kwan** [comment]
Not quite sure why "Presence Validator / Presence Validator (pull_request)" is failing when it passed on my previous commits. I don't know if it's an issue with my code or with the check. Ran `npm run compileTools` and `node --enable-source-maps tools/auto/presenceValidator.js pull_request` locally and they worked. 

image

File: websites/S/Stake.us/presence.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+presence.on("UpdateData", async () => {
+	const [showBal, showTimestamp, showCurrentGame] = await Promise.all([
+			presence.getSetting<boolean>("showBal"),
+			presence.getSetting<boolean>("showTimestamp"),
+			presence.getSetting<boolean>("showCurrentGame"),
+		]),
+		presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/9jDaQ1b.png",
+		},
+		{ pathname, search } = window.location;
```
Comment (theusaf) [inline websites/S/Stake.us/presence.ts:15]: For consistency across presences, use `document.location`
```suggestion
		{ pathname, search } = document.location;
```

**Review by theusaf — CHANGES_REQUESTED** [review]
Also, since you have settings, add a screenshot of the settings in the extension

**Comment by Kevin-Kwan** [comment]
> Also, since you have settings, add a screenshot of the settings in the extension

Thanks, I included a screenshot of the settings as the first Screenshot.

**Review by theusaf — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**